### PR TITLE
Document in the tagline dat lat_0 is required for labrd

### PR DESCRIPTION
**Pull request by rhuijben**
Add 'lat_0=' to the labrd info lines to document that the 'lat_0' argument is needed for a minimal test on this projection.

All other projections list their required arguments on this line with only this exception. (Makes this easier to test for in the SharpProj unittests, where I now have an hardcoded exception)

### CODE REVIEW

File: src/projections/labrd.cpp [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #include "proj.h"
 #include "proj_internal.h"
 
-PROJ_HEAD(labrd, "Laborde") "\n\tCyl, Sph\n\tSpecial for Madagascar";
+PROJ_HEAD(labrd, "Laborde") "\n\tCyl, Sph\n\tlat_0= Special for Madagascar";
```
Comment (rouault) [inline src/projections/labrd.cpp]: Perhaps ?
```suggestion
PROJ_HEAD(labrd, "Laborde") "\n\tCyl, Sph\n\tSpecial for Madagascar\n\tlat_0=";
```

**Review by rouault — COMMENTED** [review]
see suggestion

File: src/projections/labrd.cpp (hunk 1 above)
Comment (rhuijben) [reply src/projections/labrd.cpp]: I'm not sure what the expected format is here...
All projections I found had 2 lines max where  the first line provided the projection info and the second line arguments and sometimes a bit of text appended to it. I don't know what happens to other users of the headers if we add a third line.

I'm fine with either form. But perhaps we should add a comment on the expected format while we are at it.

**Comment by rouault** [comment]
I'm not sure either of what the expected format is (most of us were hardly born when this was devised), and if applications are supposed to rely on that. My quick search couldn't spot an example where there would have a line with parameters + free text, hence what I suggested

**Comment by rhuijben** [comment]
> My quick search couldn't spot an example where there would have a line with parameters + free text, hence what I suggested

tmerc.cpp has
PROJ_HEAD(utm, "Universal Transverse Mercator (UTM)") "\n\tCyl, Ell\n\tzone= south approx";

**Comment by rouault** [comment]
south and approx are actually boolean flags, e.g. "+proj=utm +zone=31 +south +approx"

**Comment by kbevers** [comment]
I don't understand the part about Madagascar, why is that needed?

`+lat_0` is an outright requirement, as seen from an instantiation of the projection without that parameter:

```
$ proj +proj=labrd
proj_create: Error 1027 (Invalid value for an argument): labrd: Invalid value for lat_0: lat_0 should be different from 0
Rel. 8.2.0, November 1st, 2021
<proj>:
projection initialization failure
cause: Invalid value for an argument
program abnormally terminated
```

but unfortunately we state in the [docs](https://proj.org/operations/projections/labrd.html) that the default value of `+lat_0` is 0. This should be adjusted, including a sentence saying that it can't be set to 0.

**Comment by rhuijben** [comment]
> I don't understand the part about Madagascar, why is that needed?

That was already there, so I assumed it should stay there... But I agree that it doesn't make much sense.

**Comment by kbevers** [comment]
>  That was already there, so I assumed it should stay there... But I agree that it doesn't make much sense.

I missed that from the diff. I assume the projection originally have been made specifically for use in Madagascar. 

I've added a commit to your branch with a clarification of the parameters in the Sphinx docs.

**Pull request merged**